src/test/scala/special/sigma/SigmaDslTest.scala

```diff
@@ -426,4 +426,15 @@ class SigmaDslTest extends PropSpec
     }
 
   }
+
+  property("Option methods equivalence") {
+    val opt: Option[Long] = ctx.dataInputs(0).R0[Long]
+    val eq = EqualityChecker(opt)
+    eq({ (x: Option[Long]) => x.get })("{ (x: Option[Long]) => x.get }")
+// TODO   eq({ (x: Option[Long]) => x.isEmpty })("{ (x: Option[Long]) => x.isEmpty }")
```

Comment:
@greenhat  Please give details about what has to be fixed for this TODO

Reply:
Done. It's not implemented yet.